sky/job/constants.py

```diff
@@ -20,3 +20,8 @@
 # length of the cluster name prefix is 25 to avoid the cluster name being too
 # long and truncated twice during the cluster creation.
 JOB_CLUSTER_NAME_PREFIX_LENGTH = 25
+
+# The version of the lib files that job/utils use. Whenever there is an API
+# change for the job/utils, we need to bump this version, so that the
+# user can be notified to update their SkyPilot version on the remote cluster.
```

Comment:
What does "user can be notified to update their SkyPilot version on the remote cluster." mean? Notified automatically by the system (where?), or by human devs?

Reply:
Ahh, that is a remnant. Now fixed.